tests/components/linkplay/conftest.py

```diff
@@ -0,0 +1,50 @@
+"""Test configuration and mocks for LinkPlay component."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+from linkplay.bridge import LinkPlayBridge, LinkPlayDevice
+import pytest
+
+HOST = "10.0.0.150"
+HOST_REENTRY = "10.0.0.66"
+UUID = "FF31F09E-5001-FBDE-0546-2DBFFF31F09E"
+NAME = "Smart Zone 1_54B9"
+
+
+@pytest.fixture
+def mock_linkplay_factory_bridge() -> Generator[AsyncMock]:
+    """Mock for linkplay_factory_bridge."""
+
+    bridge = AsyncMock(spec=LinkPlayBridge)
+    bridge.endpoint = "192.168.0.150"
+    bridge.device = AsyncMock(spec=LinkPlayDevice)
+    bridge.device.uuid = UUID
+    bridge.device.name = NAME
+
+    with patch(
+        "linkplay.discovery.linkplay_factory_bridge",
+        return_value=bridge,
+    ) as factory:
+        yield factory
+
+
+@pytest.fixture
+def mock_linkplay_factory_bridge_empty() -> Generator[AsyncMock]:
+    """Mock for linkplay_factory_bridge."""
+
+    with patch(
+        "linkplay.discovery.linkplay_factory_bridge",
```

Comment:
We should only patch libraries on where we use them

Reply:
This has been solved in lastest commit and was the root cause of the error I was experiencing 😄